bin/regioninfoDatEditor.ts

```diff
@@ -0,0 +1,50 @@
+import os from 'os';
+import fs from 'fs';
+import dns from 'dns';
+import util from 'util';
+
+const lookup = util.promisify(dns.lookup);
+
+export class regioninfoDatEditor {
+  regioninfoDat: string;
+  constructor() {
+    let home = os.homedir();
+    if (fs.existsSync(home + "/AppData/LocalLow/Innersloth/Among Us/regioninfo.dat"))
```

Comment:
small nit, but you're better off using `path.join()` here when concatenating file paths

Reply:
I think this one last change would be all that's needed before we merge

Reply:
sweet, i'll go ahead and do that. should i also look into dns hostname resolving? shouldn't take me too long, if we're interested in that

Reply:
I'm okay with the dns lookup happening when creating the `RegionList` object, but I don't know if it should be done via `dns.lookup()` or `dns.resolve()`, or if we need to check if it's *already* an IP before doing the lookup.

Reply:
im not well versed on the subject, but a stackoverflow search recommends `dns.resolve()`: https://stackoverflow.com/questions/40985367/node-js-dns-resolve-vs-dns-lookup. as far as checking if it's already an IP before doing the lookup, that's nothin some good ol regex can't handle

Reply:
I'm not really concerned about how we check if it's already an IP, rather if *we* need to or if the `dns` method does it for us. It doesn't really matter in the end but I still like to avoid doing unnecessary checks like that. For now it's probably fine to just pass it in to `resolve` without checking.

Reply:
welp, turns out `dns.resolve()` won't even accept an ip, for example if i put in "127.0.0.1" it throws an error, but if i put in "localhost" it returns "127.0.0.1", so it looks like we'll have to either use a regex or use `dns.lookup()`

Reply:
just made two commits, one is for the path.join fix and one is the dns lookup thing

Reply:
Nice, both IP and domain worked for me